raiden/network/rpc/client.py

```diff
@@ -772,6 +772,9 @@ def deposit(self, our_address, amount):  # pylint: disable=unused-argument
 
         log.info('deposit called', contract=pex(self.address), amount=amount)
 
+    def transferredAmount(self, participant_address):
```

Comment:
I thought we agreed on having `camelCase` for anything belonging to Solidity so that we can easily distinguish when we make calls to contracts. Think that this should only apply to the contract proxy and not the rpc client class?